x-pack/plugins/fleet/server/routes/agent/handlers.ts

```diff
@@ -182,7 +182,7 @@ export const getAgentsHandler: RequestHandler<
       kuery: request.query.kuery,
       sortField: request.query.sortField,
       sortOrder: request.query.sortOrder,
-      getTotalInactive: true,
+      getTotalInactive: request.query.showInactive,
```

Comment:
`totalInactive` should only be returned if `inactive` filter is added, otherwise bulk actions are disabled [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/bulk_actions.tsx#L77-L80)